code/datums/datacore.dm

```diff
@@ -146,7 +146,8 @@ proc/get_id_photo(var/mob/living/carbon/human/H)
 		if(E.status & ORGAN_PEG)
 			o_icobase='icons/mob/human_races/o_peg.dmi'
 		else if(E.status & ORGAN_ROBOT)
-			o_icobase='icons/mob/human_races/o_robot.dmi'
+			var/limbTier = E.limbTier
+			o_icobase="icons/mob/human_races/robotic/o_robot_[limbTier].dmi"
```

Comment:
It's kinda hard to tell considering the current robotic limbs are just black blobs, but I couldn't see a fault with it when I tested it